core/vm/contracts.go

```diff
@@ -123,7 +110,7 @@ func ActivePrecompiles(rules params.Rules) []common.Address {
 	case rules.IsByzantium:
 		return PrecompiledAddressesByzantium
 	default:
-		return PrecompiledAddressesHomestead
+		return PrecompiledAddressesBerlin
```

Comment:
why is Berlin the default ?

Reply:
Because Homestead was removed, one of the forks has to be the default. I just chose Berlin because it's at the top and seems to be the latest.

Reply:
oh cool. thanks for clarifying.